boards/oct/osd32mp1_brk/osd32mp1_brk_defconfig

```diff
@@ -0,0 +1,20 @@
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable uart driver
```

Comment:
Reformat file in english for comments, first letter capitalisations, UART not uart, etc.

Reply:
Formatting should be better now, thanks for spotting this